pkg/hubble/parser/threefour/parser.go

```diff
@@ -157,7 +157,7 @@ func (p *Parser) Decode(data []byte, decoded *pb.Flow) error {
 	defer p.packet.Unlock()
 
 	// Since v1.1.18, DecodeLayers returns a non-nil error for an empty packet, see
-	// https://github.com/google/gopacket/issues/846
+	// https://github.com/gopacket/gopacket/issues/846
```

Comment:
I think this should be left as is, as it is referring to an issue on the original repository that doesn't exist on the new repository.

Reply:
yes, thank you